src/vue/backend/src/views/guided-setup/Confirmation.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <idx-fullscreen customClass="gs" @click="closeDialog">
```

Comment:
Is this click anywhere on the page? I am not able to close it that way when I pull down this branch

Reply:
changed to a timeout